x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -1930,6 +1935,49 @@ var security = (function () {
         }
     }
 
+    var copyTargetUserToEffective = new processor.Chain()
+        .Convert({
+            fields: [
+                {from: "winlog.event_data.TargetUserSid", to: "user.effective.id"},
+                {from: "winlog.event_data.TargetUserName", to: "user.effective.name"},
+                {from: "winlog.event_data.TargetDomainName", to: "user.effective.domain"},
+            ],
+            ignore_missing: true,
+        })
+        .Add(function(evt) {
+            var user = evt.Get("winlog.event_data.TargetUserName");
+            if (user) {
+                if (/.@*/.test(user)) {
```

Comment:
The regular expression `/.@*/` means any character, followed by zero or more @ characters, so it matches all strings except the empty string. 

I think you mean "has a @" but not "starts with @", maybe you can do:
```suggestion
                if (user.indexOf('@')>0) {
```